classroom.rb

```diff
@@ -0,0 +1,60 @@
+# Represents a classroom with attributes such as label and students
+class Classroom
+  attr_accessor :label, :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    students << student
+    student.classroom = self
+  end
+end
+
+# Represents a student with attributes name and classroom.
+class Student
+  attr_accessor :name, :classroom
+
+  def initialize(name)
+    @name = name
+    @classroom = nil
+  end
+end
```

Comment:
- [ ] In the `student.rb` file after removing the `Student` class that you created in this file, kindly create a `@classroom` variable that would represent the `Classroom` object that an instance of a `Student` class will belong to in the Student class that you created. Remember that a `Classroom` instance can have many students while a `Student` instance should belong to a `Classroom` instance. 💻 🖱️ 

Please do not initialize the `@classroom` variable to have the value `nil`. It should be initialized to have the value of the classroom object being passed as a parameter in the `Student` class 👍 😎

Reply:
@Gambit142 done